### PR TITLE
Add Alert Component

### PR DESCRIPTION
**Pull request by kavyakunder**
image
image
imageVar Varients of alert component includes simple alert, alert with outline, alerts with close button.

### CODE REVIEW

File: Components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
+@import "../../commonFiles/navbar.css";
+@import "../../commonFiles/sidebar.css";
+@import "../../commonFiles/commonStyle.css";
+.alerts-container {
```
Comment (Hamdrive) [inline Components/Alert/alert.css]: Give a new line spacing between imports and properties

File: Components/Alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  color: white;
+  border-radius: 4px;
+  padding: 1rem 1.5rem;
+  margin: 1rem 0;
+  font-size: 1.1rem;
+}
+/* primary alert */
+.alert-primary {
+  background-color: #cfe2ff;
+  color: #084fb9;
```
Comment (Hamdrive) [inline Components/Alert/alert.css]: Preferable to store the colors in a separate CSS file, access them using `var()`

**Review by Hamdrive — COMMENTED** [review]
Looks good, very clean code

File: Components/Alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  position: absolute;
+  right: 0;
+  top: 10px;
+  margin-right: 10px;
+}
+.close-btn {
+  border: 1px solid darkgray;
+  padding: 20px;
+  color: darkgray;
+}
```
Comment (ZuberDunge) [inline Components/Alert/alert.css:82]: Use color variables instead of color names and hex codes.
you can store all colors in base.css

**Review by ZuberDunge — COMMENTED** [review]
looks good tm.

File: Components/Alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  border-radius: 10px;
+}
+.alerts {
+  font-size: 1.3rem;
+  width: 100%;
+  color: white;
+  border-radius: 4px;
+  padding: 1rem 1.5rem;
+  margin: 1rem 0;
+  font-size: 1.1rem;
```
Comment (prachi-sahani) [inline Components/Alert/alert.css:18]: you have set the value of font size twice

**Review by prachi-sahani — COMMENTED** [review]
Code looks fine! 👍🏻 
Minor feedback - add some space between the icon and the text

File: Components/Alert/alert.css (hunk 1 above)
Comment (kavyakunder) [reply Components/Alert/alert.css]: I have made the required changes, please do have a look.

File: Components/Alert/alert.css (hunk 3 above)
Comment (kavyakunder) [reply Components/Alert/alert.css:82]: I have made the required changes, please do have a look.

File: Components/Alert/alert.css (hunk 4 above)
Comment (kavyakunder) [reply Components/Alert/alert.css:18]: I have removed double assignment of font-size.

**Pull request merged**